components/infobox/wikis/dota2/infobox_league_custom.lua

```diff
@@ -26,7 +26,6 @@ local _GAMES = {
 	dota2 = {name = 'Dota 2', category = 'Dota 2 Competitions'},
 	dota = {name = 'DotA', category = 'DotA Competitions'},
 	hon = {name = 'Heroes of Newerth', category = 'Heroes of Newerth Competitions'},
-	['auto chess'] = {name = 'Auto Chess', category = 'Auto Chess Competitions'},
```

Comment:
this removal intended?

Reply:
Yes, there's only one tournament on the wiki, and it's supposed to get moved to its own wiki.
https://discord.com/channels/93055209017729024/213718409567928331/976450350598979634